index.html

```diff
@@ -34,5 +34,175 @@ <h1 class="greetings"> Hey there <br>
+                  <div class="button1"><button class="button-text" type="button">See project </button></div>
+                </div>
+            </div>
+            <div  class="placeholder-container">
+                <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+                <div class="text-container">
+                  <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+                  <ul class="text">
+                    <li class="text-ruby">Ruby on Rails</li>
+                    <li class="text-css">CSS</li>
+                    <li class="text-java">JavaScript</li>
+                    <li class="text-html">HTML</li>
+                  </ul>
+                  <div class="button1"><button class="button-text" type="button">See project </button></div>
+                </div>
+            </div>
+            <div  class="placeholder-container">
+                <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+                <div class="text-container">
+                  <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+                  <ul class="text">
+                    <li class="text-ruby">Ruby on Rails</li>
+                    <li class="text-css">CSS</li>
+                    <li class="text-java">JavaScript</li>
+                    <li class="text-html">HTML</li>
+                  </ul>
+                  <div class="button1"><button class="button-text" type="button">See project </button></div>
+                </div>
+            </div>
+            <div  class="placeholder-container">
+              <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+              <div class="text-container">
+                <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+                <ul class="text">
+                  <li class="text-ruby">Ruby on Rails</li>
+                  <li class="text-css">CSS</li>
+                  <li class="text-java">JavaScript</li>
+                  <li class="text-html">HTML</li>
+                </ul>
+                <div class="button1"><button class="button-text" type="button">See project </button></div>
+              </div>
+          </div>
+          </section>
+            <section class="about-container">
+              <div class="text-aline">
+                </div>
+                <div class="text-about">
+                  <div class="plane-container">
+                    <h3>About me</h3>
+                    <div class="plane-image">
+                      <img src="images/plane.svg" alt="plane">
+                    </div>
+                  </div>
+                  
+                    <div>
+                      <p class="h6"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
```

Comment:
- [ ] Please apply the correct font family to this text. It will look better as the font will match the design. 👍🏾